### DISCUSSION TITLE
💡 Add backward key navigation for options

### DISCUSSION THREAD

**Discussion by harshsinghatz** [monkeytypegame/monkeytype#2544] [Ideas] [answered]
After the typing practice, we get a graph of our performance and multiple options ( such as Next test, Repeat test, Practice words, etc ) which we can navigate forward through the "tab" button but there is no way of navigating backward.

### My Recommendations for the keys:
- "Backspace" button
- "shift" button

### Why should we implement this?
- As monkeytype is typing oriented application it is very suitable 
- It's easier for the typist to navigate

**Comment by Miodec** [comment, marked as answer]
I believe shift + tab is the default in every browser for moving focus backwards.

**Comment by harshsinghatz** [comment]
My bad didn't know about that 🙌
